### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by Will-Hsu**
Worked on the LandingPage component to reflect the updated Homehub landing page design on Figma. The page should be responsive to mobile view as well.

### CODE REVIEW

**Review by bamit5 — CHANGES_REQUESTED** [review]
Can you review the changes i specified in the comments on the code?

File: FrontEnd/src/pages/index.tsx [hunk 1]
```diff
@@ -1,81 +1 @@
...
-    <Row className={styles.center}>
-      <Button variant="secondary" href="https://forms.gle/jxmNTJE6L2dZBsug9">
-        <div className={styles.buttonInner}>Sign up FOR FREE</div>
-      </Button>
-    </Row>
-  </Container>
-);
-
-export default Landing;
+export { default } from '../components/LandingPage';
```
[the remark this answers is not in the document]
Comment (Will-Hsu) [reply FrontEnd/src/pages/index.tsx:1]: Just updated!

File: FrontEnd/src/components/LandingPage/LandingPage.module.scss [hunk 2]
```diff
@@ -0,0 +1,120 @@
+  .title{
```
[the remark this answers is not in the document]
Comment (Will-Hsu) [reply FrontEnd/src/components/LandingPage/LandingPage.module.scss:1]: Noted! I will be sure to update them once we have reuseable variables!

File: FrontEnd/src/components/LandingPage/LandingPage.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
+import React, { FunctionComponent } from 'react';
+import { Button, Col, Container, Row } from 'react-bootstrap';
+import { landingIcons } from '@icons';
+import styles from './LandingPage.module.scss';
+
+const Landing: FunctionComponent = () => (
```
[the remark this answers is not in the document]
Comment (Will-Hsu) [reply FrontEnd/src/components/LandingPage/LandingPage.tsx]: Good idea! I split the page into 3 main components: WelcomeInfo, Why-Homehub, and Landing.

**Comment by Will-Hsu** [comment]
Thanks for the review! I made fixes to address your three feedback. Let me know if there are any changes to be done! 
Changes: 1. Use @components/LandingPage for consistency in the codebase. 2. Refactor the Landing Page code into 3 components to increase code readability.

**Comment by Will-Hsu** [comment]
The changes have been incorporated to the master branch under a different pull request -- May 29.

**Pull request closed without merging**